x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/metrics.tsx

```diff
@@ -76,7 +76,6 @@ function buildMetricOperation<T extends MetricColumn<string>>({
     },
```

Comment:
There is a problem in this line: https://github.com/elastic/kibana/pull/94710/files#diff-e3c30f7e1cc23b9be6b4b2d689290e96925f48369dfcaa44d641014dfdc7bc80R68

The aggregation restriction mapping logic in https://github.com/elastic/kibana/blob/52a1ce17233079138ee4cf6bd9f86eaa6266ea7a/x-pack/plugins/lens/public/indexpattern_datasource/loader.ts#L105 is using the Elasticsearch aggregation names, but now they are not in sync anymore (`avg` vs `average`). Aggregation restrictions for the average operation won't be loaded correctly anymore. We have to check aggregation restrictions for `avg` instead of `average`.